clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -6480,20 +6480,23 @@ SanitizerOrdinalToCheckLabel(SanitizerKind::SanitizerOrdinal Ordinal) {
 llvm::DILocation *CodeGenFunction::SanitizerAnnotateDebugInfo(
     ArrayRef<SanitizerKind::SanitizerOrdinal> Ordinals,
     SanitizerHandler Handler) {
+  llvm::DILocation *CheckDI = Builder.getCurrentDebugLocation();
```

Comment:
Nit: variable naming - "CheckDI" is a DILocation, not DebugInfo like the similarly named "DI".

Reply:
done